src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/limitedpartner/dao/LimitedPartnerDataDao.java

```diff
@@ -1,24 +1,127 @@
 package uk.gov.companieshouse.limitedpartnershipsapi.model.limitedpartner.dao;
 
+import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.data.mongodb.core.mapping.Field;
-import uk.gov.companieshouse.limitedpartnershipsapi.model.limitedpartner.LimitedPartnerType;
+import uk.gov.companieshouse.limitedpartnershipsapi.model.common.dao.AddressDao;
+import uk.gov.companieshouse.limitedpartnershipsapi.model.limitedpartner.Currency;
 
+import java.time.LocalDate;
+
+@Document(collection = "limited_partners")
```

Comment:
for General partner the @Document goes on the GeneralPartnerDao as it contains the GeneralPartnerDataDao. So i think similar should happen here, the parent Dao has the @Document and and classes used by that don't need @Document.